sensor/common/image/service_impl.go

```diff
@@ -78,9 +86,19 @@ func (s *serviceImpl) GetImage(ctx context.Context, req *sensor.GetImageRequest)
 		}, nil
 	}
 
-	img, err := s.localScan.EnrichLocalImage(ctx, s.centralClient, req.GetImage())
+	var err error
+	var img *storage.Image
+	if req.GetImage().GetIsClusterLocal() {
+		img, err = s.localScan.EnrichLocalImage(ctx, s.centralClient, req.GetImage())
+	} else {
+		// ForceLocalImageScanning must be true
+		img, err = s.localScan.EnrichLocalImageInNamespace(ctx, s.centralClient, req.GetImage(), req.GetDeployment().GetNamespace())
```

Comment:
From this call, I think we just need the deployment's namespace, so let's update the proto to just give the namespace instead of the deployment struct

Reply:
will do